.github/workflows/ci.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     services:
       mysql:
-        image: mysql/mysql-server:8.0.30
```

Comment:
why was this downgraded?

Reply:
mysql 8 uses `caching_sha2_password` by default which is currently unsupported by Trilogy

Reply:
It still possible to make trilogy work with mysql 8 by forcing the legacy auth mode.